src/rtc.c

```diff
@@ -0,0 +1,518 @@
+/**
+ * @file rtc.c
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
+
+#include "libdragon.h"
+
+/**
+ * @defgroup rtc Real-Time Clock Subsystem
+ * @ingroup libdragon
+ * @brief Joybus real-time clock interface.
+ *
+ * The Joybus real-time clock is an accessory that was only included on
+ * one official cartridge that was only available in Japan. The Joybus RTC
```

Comment:
IIUC, it was an official option supported by Nintendo, implemented in PIF firmware, but only animal crossing decided to use it in its cartridge. I'd specify this, because otherwise it sounds like a game-specific hardware extension which would have very little relevance

Reply:
I'd also specify here in this first paragraph that it's supported by some development kits (64drive / ed64) and some emulators (list here which)

Reply:
@jago85 is the RTC actually a feature that was always intended to be supported by the PIF, or is it a game-specific convention that's mostly implemented on the cartridge circuitry? From what I can tell, it behaves a lot like EEPROM (RTC is on the same PIF channel as EEPROM) but responds to different command identifiers.

Reply:
Oh, I really don't know when they were planning this. Your question is about the original PIF, isn't it? 
Yes, the RTC is completely implemented inside the cart, just using the SI to transport the data. The PIF doesn't care what is connected to the bus and what the commands are good for. It just shifts the data out to and in from one of the five SI channels. So from the PIF's point of view, the interface is pretty generic.
I've heared that Animal Forest was originally planned for the 64DD. As it flopped they ported the game to cartridge and needed a replacement RTC. So they designed an RTC chip with SI interface. And it was designed with the ability to coexist with an EEPROM chip on the same bus.